bndtools.m2e/lifecycle-mapping-metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<lifecycleMappingMetadata>
+	<pluginExecutions>
+		<pluginExecution>
+			<pluginExecutionFilter>
+				<groupId>biz.aQute.bnd</groupId>
+				<artifactId>bnd-maven-plugin</artifactId>
+				<versionRange>[3.0.0,)</versionRange>
+				<goals>
+					<goal>bnd-process</goal>
+				</goals>
+			</pluginExecutionFilter>
+			<action>
+				<configurator>
+					<id>bndtools.m2e.bndconfigurator</id>
+				</configurator>
+			</action>
+		</pluginExecution>
+	</pluginExecutions>
+</lifecycleMappingMetadata>
```

Comment:
This commit puts a file in src so it is non-empty and thus does not need a .gitignore to keep it non-empty. I understand the use of empty .gitignore files to keep a folder non-empty, but this is not the case here.